CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 
 ## Bug Fixes
+* Fixed SpikeInterface physical unit properties (`gain_to_physical_unit`, `offset_to_physical_unit`, `physical_unit`) being incorrectly added to electrodes table instead of being handled in ElectricalSeries.
```

Comment:
Format this in the same way as all of the other bullet points in this changelog file, referencing the PR that it is associated with.